firefox-ios/Shared/Strings.swift

```diff
@@ -7766,7 +7766,7 @@ extension String {
             public static let FileCountText = MZLocalizedString(
                 key: "LiveActivity.Downloads.FileCountText.v137",
                 tableName: "LiveActivity",
-                value: "Downloading %@ files",
+                value: "Downloading Files: %@",
                 comment: "Displayed during a download in Live Activity or Dynamic Island. The %@ placeholder represents the count of multiple files (e.g. Downloading 2 files).")
```

Comment:
Awesome, can we update the comment to reflect the new string as well?

```suggestion
                value: "Downloading Files: %@",
                comment: "Displayed during a download in Live Activity or Dynamic Island. The %@ placeholder represents the count of multiple files (e.g. Downloading Files: 2).")
```